webapp/loggedin/modules/users/userList/userListView.scss

```diff
@@ -1,5 +1,25 @@
-.user-list {
-  .table__row, .table__row-header {
+@import "../../../../style/vars";
+
+.users-list {
+  .table__rows {
+    cursor: pointer;
```

Comment:
this is a generic behavior of the table component.
the style of the row hover should go to table.scss.
also, each single row should have the hover style

Reply:
ok